File: src/ProjectR.Scripting/content/scripts/afflictions/Passives/SwiftFeet.lua
function GetName()
  return "Swift Feet"
end

function GetType()
  return Passive
end

function GetHookPoints()
  return { RollingEvasion }
end

function OnAttachment(char)
end

function OnRemoval(char)
end

function Trigger(hookPoint, char)
  if hookPoint == RollingEvasion then
    param1 = param1 * 1.1
  end
end

File: src/ProjectR.Scripting/content/scripts/afflictions/Passives/SwiftFeet.cs
using ProjectR.Interfaces.Model;

namespace ProjectR.Scripting.Afflictions
{
    public class SwiftFeet : Affliction
    {
        public override string Name { get { return "Swift Feet"; } }

        public override AfflictionType Type { get { return AfflictionType.Passive; } }

        protected override HookPoint[] HookPoints
        {
            get
            {
                return new[]
                {
                    HookPoint.RollingEvasion, 
                };
            }
        }

        protected override void OnRollingEvasion(ICharacter character, ref double evaChance)
        {
            evaChance *= 1.1;
        }
    }
}